### ISSUE TITLE
Report generation fails

### ISSUE THREAD

**Issue by ThaStealth** [allure-framework/allure-teamcity-plugin#32]
Allure 1.4.19
Teamcity-Plugin 2.2
I get the following exception:

```
[Step 7/7] Starting: D:\TeamCity\BuildAgent\jre\bin\java.exe -Dallure.config=D:\TeamCity\BuildAgent\temp\buildTmp\allure-config3745144478600058394\allure.properties -Dallure.home=D:\TeamCity\BuildAgent\tools\allure-commandline -classpath D:\TeamCity\BuildAgent\tools\allure-commandline\lib/*;D:\TeamCity\BuildAgent\tools\allure-commandline\conf ru.yandex.qatools.allure.CommandLine -v generate D:\TeamCity\BuildAgent\work\8591396e50c5baa5\results --output D:\TeamCity\BuildAgent\temp\buildTmp\allure-report1383974199034739206
[09:22:12][Step 7/7] in directory: D:\TeamCity\BuildAgent\work\8591396e50c5baa5
[09:22:14][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.MigrationPlugin@1f4931d]
[09:22:14][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.XUnitPlugin@145df59]
[09:22:14][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.TimelinePlugin@1b9f5a4]
[09:22:14][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.BehaviorsPlugin@10f7aa]
[09:22:14][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.DefectsPlugin@4970b0]
[09:22:14][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.GraphPlugin@12ff811]
[09:22:19][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.EnvironmentPlugin@f37659]
[09:22:19][Step 7/7] [main] DEBUG ru.yandex.qatools.allure.data.utils.ServiceLoaderUtils - Found Plugin [ru.yandex.qatools.allure.data.plugins.TotalPlugin@cb8df7]
[09:22:20][Step 7/7] Exception in thread "main" javax.xml.bind.DataBindingException: javax.xml.bind.UnmarshalException
[09:22:20][Step 7/7]  - with linked exception:
[09:22:20][Step 7/7] [org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.]
[09:22:20][Step 7/7] 	at javax.xml.bind.JAXB.unmarshal(JAXB.java:245)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.commons.AllureFileUtils.unmarshal(AllureFileUtils.java:59)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.commons.AllureFileUtils.unmarshal(AllureFileUtils.java:50)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.commons.AllureFileUtils.unmarshal(AllureFileUtils.java:40)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.data.io.TestSuiteReader$TestSuiteResultIterator.next(TestSuiteReader.java:48)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.data.io.TestSuiteReader$TestSuiteResultIterator.next(TestSuiteReader.java:1)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.data.io.TestCaseReader$TestCaseResultIterator.nextSuite(TestCaseReader.java:43)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.data.io.TestCaseReader$TestCaseResultIterator.hasNext(TestCaseReader.java:53)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.data.AllureReportGenerator.generate(AllureReportGenerator.java:57)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.data.AllureReportGenerator.generate(AllureReportGenerator.java:53)
[09:22:20][Step 7/7] 	at ru.yandex.qatools.allure.AllureMain.main(AllureMain.java:48)
[09:22:20][Step 7/7] Caused by: javax.xml.bind.UnmarshalException
[09:22:20][Step 7/7]  - with linked exception:
[09:22:20][Step 7/7] [org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.]
[09:22:20][Step 7/7] 	at javax.xml.bind.helpers.AbstractUnmarshallerImpl.createUnmarshalException(AbstractUnmarshallerImpl.java:335)
[09:22:20][Step 7/7] 	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.createUnmarshalException(UnmarshallerImpl.java:563)
[09:22:20][Step 7/7] 	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:249)
[09:22:20][Step 7/7] 	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:221)
[09:22:20][Step 7/7] 	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:276)
[09:22:20][Step 7/7] 	at javax.xml.bind.JAXB.unmarshal(JAXB.java:242)
[09:22:20][Step 7/7] 	... 10 more
[09:22:20][Step 7/7] Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:203)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:177)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:441)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:368)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1437)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next(XMLDocumentScannerImpl.java:999)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:117)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:510)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:848)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:777)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1213)
[09:22:20][Step 7/7] 	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:649)
[09:22:20][Step 7/7] 	at com.sun.xml.internal.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:243)
[09:22:20][Step 7/7] 	... 13 more
[09:22:20][Step 7/7] ERROR: Command aborted due to exception org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1).
[09:22:20][Step 7/7] Process exited with code 1



[09:22:20][Step 7/7] Step Reports (Allure Report) failed
```

Contents of the xml file:

```xml
<?xml version="1.0" encoding="utf-8"?><test-suite xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" start="1444806780831" stop="1444806800338" version="1.4.0" xmlns="urn:model.allure.qatools.yandex.ru"><name xmlns="" /><test-cases xmlns=""><test-case start="1444806788324" stop="1444806788332" status="passed"><name>CalculateMd5Test_2</name></test-case><test-case start="1444806785352" stop="1444806785381" status="passed"><name>CreateMultiAssetDataReportTest</name></test-case><test-case start="1444806788227" stop="1444806788239" status="passed"><name>ArePositionsEqualTest</name></test-case><test-case start="1444806797907" stop="1444806797928" status="passed"><name>LogEventTest</name></test-case><test-case start="1444806798061" stop="1444806798455" status="passed"><name>LoadSaveDataBinaryTest_1</name></test-case><test-case start="1444806788196" stop="1444806788209" status="passed"><name>StoreFlashDownloadDataTest</name></test-case><test-case start="1444806793443" stop="1444806793450" status="passed"><name>GetTimeZoneInfoFromServerTimeZoneIndexTest</name></test-case><test-case start="1444806788167" stop="1444806788182" status="passed"><name>PowerModeTest</name></test-case><test-case start="1444806798720" stop="1444806798729" status="passed"><name>TransmitTimestampTest</name></test-case><test-case start="1444806788148" stop="1444806788156" status="passed"><name>ImplodeDigitalSensorIndexTest</name></test-case><test-case start="1444806785314" stop="1444806785349" status="passed"><name>CreateExceptionDataReportTest_2</name></test-case><test-case start="1444806792871" stop="1444806792879" status="passed"><name>FirmwareStringToIntegerTest</name></test-case><test-case start="1444806787872" stop="1444806787914" status="passed"><name>GetDeviceFirmwaresTest_3</name></test-case><test-case start="1444806799413" stop="1444806799425" status="passed"><name>SetupClientListenerTest_1</name></test-case><test-case start="1444806783573" stop="1444806783614" status="passed"><name>CreateDataFieldsToUseFilterFromLeadingTypesTest</name></test-case><test-case start="1444806785767" stop="1444806785796" status="passed"><name>ParseDataIntoLinearDataTest_2</name></test-case><test-case start="1444806788400" stop="1444806788410" status="passed"><name>ConvertToDateTimeSeconds</name></test-case><test-case start="1444806788131" stop="1444806788139" status="passed"><name>GetTotalAlarmValuesTest</name></test-case><test-case start="1444806793453" stop="1444806793465" status="passed"><name>GetTimeZoneNameIfAvailableTest</name></test-case><test-case start="1444806785724" stop="1444806785766" status="passed"><name>ParseDataIntoLinearDataTest_1</name></test-case><test-case start="1444806787794" stop="1444806787803" status="passed"><name>CreateDataFileTest_1_Failure</name></test-case><test-case start="1444806785470" stop="1444806785553" status="passed"><name>LoadAssetInfoTest</name></test-case><test-case start="1444806793477" stop="1444806793485" status="passed"><name>GetValidControlColorTest</name></test-case><test-case start="1444806792946" stop="1444806792960" status="passed"><name>GetFilesFromFolderTest</name></test-case><test-case start="1444806798655" stop="1444806798665" status="passed"><name>OriginateTimestampTest</name></test-case><test-case start="1444806797719" stop="1444806797727" status="passed"><name>StringToBytesTest</name></test-case><test-case start="1444806792972" stop="1444806792977" status="passed"><name>GetFirmwareVersionFromFileNameTest</name></test-case><test-case start="1444806797761" stop="1444806797770" status="passed"><name>TimeSpanToStringShortTest</name></test-case><test-case start="1444806788388" stop="1444806788398" status="passed"><name>CharToBoolTest_2_Failure</name></test-case><test-case start="1444806797711" stop="1444806797717" status="passed"><name>SafeToStringTest</name></test-case><test-case start="1444806797839" stop="1444806797868" status="passed"><name>ClearFileTest</name></test-case><test-case start="1444806788272" stop="1444806788281" status="passed"><name>BytesToStringTest</name></test-case><test-case start="1444806797827" stop="1444806797837" status="passed"><name>AddThreadTest</name></test-case><test-case start="1444806797784" stop="1444806797799" status="passed"><name>UriCombineTest</name></test-case><test-case start="1444806797929" stop="1444806797944" status="passed"><name>LogExceptionTest</name></test-case><test-case start="1444806797690" stop="1444806797699" status="passed"><name>RoundToHoursTest</name></test-case><test-case start="1444806798482" stop="1444806798491" status="passed"><name>LoadSaveDataBinaryTest_Failure</name></test-case><test-case start="1444806788121" stop="1444806788130" status="passed"><name>GetTotalAlarmsListTest</name></test-case><test-case start="1444806792904" stop="1444806792911" status="passed"><name>FloorToHoursTest</name></test-case><test-case start="1444806785596" stop="1444806785596" status="passed"><name>LoadFridgeDataTest_1</name></test-case><test-case start="1444806788069" stop="1444806788076" status="passed"><name>GetMenuStructureDataTest</name></test-case><test-case start="1444806788211" stop="1444806788225" status="passed"><name>StoreLoggingBufferTest</name></test-case><test-case start="1444806780831" stop="1444806783546" status="passed"><name>CreateAssetDataRowFromDataRowTest</name></test-case><test-case start="1444806792853" stop="1444806792861" status="passed"><name>FeetToMetersTest</name></test-case><test-case start="1444806787915" stop="1444806787932" status="passed"><name>GetDeviceMenuStructuresTest</name></test-case><test-case start="1444806785633" stop="1444806785659" status="passed"><name>LoadFridgeDiagnosticDataTest_2</name></test-case><test-case start="1444806798668" stop="1444806798680" status="passed"><name>ReceiveTimestampTest</name></test-case><test-case start="1444806788185" stop="1444806788193" status="passed"><name>RunModeTest</name></test-case><test-case start="1444806788454" stop="1444806788459" status="passed"><name>ConvertToJulianSecondsTest</name></test-case><test-case start="1444806787805" stop="1444806787822" status="passed"><name>CreateDataFileTest_2</name></test-case><test-case start="1444806787836" stop="1444806787842" status="passed"><name>CreateDebugFileTest_Failure</name></test-case><test-case start="1444806792914" stop="1444806792922" status="passed"><name>FormatBytesTest</name></test-case><test-case start="1444806788263" stop="1444806788270" status="passed"><name>BytesToHexTest</name></test-case><test-case start="1444806788467" stop="1444806788491" status="passed"><name>CopyStreamTest</name></test-case><test-case start="1444806792986" stop="1444806793278" status="passed"><name>GetIndexOfTimeZoneInfoTest</name></test-case><test-case start="1444806799465" stop="1444806799475" status="passed"><name>StartClientListenerTest_Failure</name></test-case><test-case start="1444806797610" stop="1444806797624" status="passed"><name>ReadFileTest</name></test-case><test-case start="1444806795578" stop="1444806795585" status="passed"><name>MinTest</name></test-case><test-case start="1444806793364" stop="1444806793398" status="passed"><name>GetNetworkLocalDirectoryTest</name></test-case><test-case start="1444806785578" stop="1444806785596" status="passed"><name>LoadEbsDataTest_2</name></test-case><test-case start="1444806785825" stop="1444806785835" status="passed"><name>ProcessSetPointsTest</name></test-case><test-case start="1444806788369" stop="1444806788377" status="passed"><name>CharToBoolTest_1</name></test-case><test-case start="1444806798043" stop="1444806798060" status="passed"><name>CheckSinglePathTest</name></test-case><test-case start="1444806788360" stop="1444806788368" status="passed"><name>CelciusToFahrenheitTest</name></test-case><test-case start="1444806795597" stop="1444806795604" status="passed"><name>MphToMphTest</name></test-case><test-case start="1444806788343" stop="1444806788350" status="passed"><name>CalculateSha1Test_2</name></test-case><test-case start="1444806793432" stop="1444806793441" status="passed"><name>GetTimeZoneIndexFromIdTest</name></test-case><test-case start="1444806798693" stop="1444806798701" status="passed"><name>ReferenceTimestampTest</name></test-case><test-case start="1444806795606" stop="1444806797608" status="passed"><name>OldMachineIdTest</name></test-case><test-case start="1444806797679" stop="1444806797688" status="passed"><name>RetrieveBytesImageFileTest</name></test-case><test-case start="1444806788352" stop="1444806788359" status="passed"><name>CeilToHoursTest</name></test-case><test-case start="1444806785705" stop="1444806785722" status="passed"><name>ParseDatabaseCommandWithUserTest</name></test-case><test-case start="1444806797888" stop="1444806797906" status="passed"><name>GetLogFilesTest</name></test-case><test-case start="1444806785662" stop="1444806785704" status="passed"><name>MergeFilterAndFieldsTest</name></test-case><test-case start="1444806785573" stop="1444806785578" status="passed"><name>LoadEbsDataTest_1</name></test-case><test-case start="1444806798581" stop="1444806798622" status="passed"><name>SaveBinaryApplicationSettingsTest_Failure</name></test-case><test-case start="1444806797773" stop="1444806797781" status="passed"><name>UppercaseFirstTest</name></test-case><test-case start="1444806793400" stop="1444806793408" status="passed"><name>GetPrecisionTest</name></test-case><test-case start="1444806792979" stop="1444806792984" status="passed"><name>GetFirmwareVersionFromFileNameTest_Failure</name></test-case><test-case start="1444806788508" stop="1444806788517" status="passed"><name>CountStringOccurencesTest</name></test-case><test-case start="1444806793555" stop="1444806793564" status="passed"><name>LocalTimeToUtcTest</name></test-case><test-case start="1444806785384" stop="1444806785410" status="passed"><name>CreateSingleAssetDataRowTest</name></test-case><test-case start="1444806788460" stop="1444806788466" status="passed"><name>ConvertToJulianTest</name></test-case><test-case start="1444806798000" stop="1444806798006" status="passed"><name>GenerateTest_ExclusionSet</name></test-case><test-case start="1444806787853" stop="1444806787864" status="passed"><name>GetDeviceFirmwaresTest_1</name></test-case><test-case start="1444806788251" stop="1444806788260" status="passed"><name>BoolToCharTest_2</name></test-case><test-case start="1444806792844" stop="1444806792851" status="passed"><name>FahrenheitToCelciusTest</name></test-case><test-case start="1444806797801" stop="1444806797809" status="passed"><name>UtcToLocalTimeTest</name></test-case><test-case start="1444806793486" stop="1444806793492" status="passed"><name>HaversineFormulaTest</name></test-case><test-case start="1444806785412" stop="1444806785454" status="passed"><name>FilterMarkersTest_1</name></test-case><test-case start="1444806798007" stop="1444806798013" status="passed"><name>GenerateTest_OneDifferenceBoundary</name></test-case><test-case start="1444806797986" stop="1444806797991" status="passed"><name>GenerateTest_ConsecutiveCharacters</name></test-case><test-case start="1444806792924" stop="1444806792933" status="passed"><name>FormatFirmwareStringTest</name></test-case><test-case start="1444806797701" stop="1444806797709" status="passed"><name>SafeResourceStrTest</name></test-case><test-case start="1444806798492" stop="1444806798522" status="passed"><name>LoadSaveDataTest_1</name></test-case><test-case start="1444806797945" stop="1444806797959" status="passed"><name>LogUserActionTest</name></test-case><test-case start="1444806798456" stop="1444806798481" status="passed"><name>LoadSaveDataBinaryTest_2</name></test-case><test-case start="1444806792828" stop="1444806792834" status="passed"><name>DecodeKeyTest_Failure</name></test-case><test-case start="1444806788158" stop="1444806788165" status="passed"><name>IsValidPositionDataTest</name></test-case><test-case start="1444806785554" stop="1444806785572" status="passed"><name>LoadDoorLockDataTest</name></test-case><test-case start="1444806788084" stop="1444806788119" status="passed"><name>GetStackTraceTest</name></test-case><test-case start="1444806798572" stop="1444806798580" status="passed"><name>SaveBinaryApplicationSettingsTest</name></test-case><test-case start="1444806788334" stop="1444806788341" status="passed"><name>CalculateSha1Test_1</name></test-case><test-case start="1444806788411" stop="1444806788419" status="passed"><name>ConvertToDateTimeTest</name></test-case><test-case start="1444806798524" stop="1444806798534" status="passed"><name>LoadSaveDataTest_Failure</name></test-case><test-case start="1444806788242" stop="1444806788249" status="passed"><name>BoolToCharTest_1</name></test-case><test-case start="1444806798553" stop="1444806798571" status="passed"><name>SaveApplicationSettingsTest_Failure</name></test-case><test-case start="1444806788030" stop="1444806788035" status="passed"><name>GetFirmwareDataTest_2</name></test-case><test-case start="1444806799845" stop="1444806800338" status="passed"><name>EmptyTestMethod</name></test-case><test-case start="1444806797749" stop="1444806797760" status="passed"><name>TimeSpanToStringLongTest</name></test-case><test-case start="1444806799426" stop="1444806799454" status="passed"><name>SetupClientListenerTest_2</name></test-case><test-case start="1444806793566" stop="1444806795542" status="passed"><name>MachineIdTest</name></test-case><test-case start="1444806795556" stop="1444806795564" status="passed"><name>MetersToFeetTest</name></test-case><test-case start="1444806783638" stop="1444806783676" status="passed"><name>CreateDataFieldsToUseFromRowTest</name></test-case><test-case start="1444806785797" stop="1444806785809" status="passed"><name>ParseDataIntoNumericDataTest</name></test-case><test-case start="1444806788379" stop="1444806788386" status="passed"><name>CharToBoolTest_2</name></test-case><test-case start="1444806785456" stop="1444806785467" status="passed"><name>FilterMarkersTest_2</name></test-case><test-case start="1444806799477" stop="1444806799833" status="passed"><name>StopClientListenerTest</name></test-case><test-case start="1444806798623" stop="1444806798653" status="passed"><name>IsResponseValidTest</name></test-case><test-case start="1444806798536" stop="1444806798551" status="passed"><name>SaveApplicationSettingsTest</name></test-case><test-case start="1444806788047" stop="1444806788067" status="passed"><name>GetFirmwareDataTest_3_Failure</name></test-case><test-case start="1444806787865" stop="1444806787871" status="passed"><name>GetDeviceFirmwaresTest_2</name></test-case><test-case start="1444806797881" stop="1444806797887" status="passed"><name>FindThreadNameTest</name></test-case><test-case start="1444806799835" stop="1444806799843" status="passed"><name>ToStringTest</name></test-case><test-case start="1444806790825" stop="1444806792827" status="passed"><name>DecodeKeyTest</name></test-case><test-case start="1444806783616" stop="1444806783635" status="passed"><name>CreateDataFieldsToUseFilterFromRowTest</name></test-case><test-case start="1444806788528" stop="1444806788537" status="passed"><name>CreateCrcTest</name></test-case><test-case start="1444806797729" stop="1444806797737" status="passed"><name>TimeSpanPercentageTest</name></test-case><test-case start="1444806797960" stop="1444806797976" status="passed"><name>GenerateTest_BigBoundary</name></test-case><test-case start="1444806792882" stop="1444806792903" status="passed"><name>FixUtcDateTimeFieldInDataTableTest</name></test-case><test-case start="1444806797738" stop="1444806797748" status="passed"><name>TimeSpanToStringHmsTest</name></test-case><test-case start="1444806798014" stop="1444806798020" status="passed"><name>GenerateTest_RepeatCharacters</name></test-case><test-case start="1444806798731" stop="1444806798739" status="passed"><name>UpdateTimeTest_Failure</name></test-case><test-case start="1444806788313" stop="1444806788322" status="passed"><name>CalculateMd5Test_1</name></test-case><test-case start="1444806788037" stop="1444806788046" status="passed"><name>GetFirmwareDataTest_3</name></test-case><test-case start="1444806793410" stop="1444806793418" status="passed"><name>GetResourceStringTest</name></test-case><test-case start="1444806785811" stop="1444806785823" status="passed"><name>ParseImportDataIntoRowDataTest</name></test-case><test-case start="1444806795587" stop="1444806795594" status="passed"><name>MphToKphTest</name></test-case><test-case start="1444806798740" stop="1444806799411" status="passed"><name>SetupClientListenerDualBindingTest</name></test-case><test-case start="1444806793501" stop="1444806793511" status="passed"><name>HexToBytesTest</name></test-case><test-case start="1444806797645" stop="1444806797677" status="passed"><name>ResizeImageToFitTest</name></test-case><test-case start="1444806788493" stop="1444806788499" status="passed"><name>CopyStreamTest_Failure_1</name></test-case><test-case start="1444806793523" stop="1444806793531" status="passed"><name>HexToLongTest</name></test-case><test-case start="1444806785289" stop="1444806785312" status="passed"><name>CreateExceptionDataReportTest_1</name></test-case><test-case start="1444806787934" stop="1444806788029" status="passed"><name>GetFirmwareDataTest_1</name></test-case><test-case start="1444806798682" stop="1444806798691" status="passed"><name>ReferenceIdTest</name></test-case><test-case start="1444806799455" stop="1444806799464" status="passed"><name>StartClientListenerTest</name></test-case><test-case start="1444806783550" stop="1444806783572" status="passed"><name>CreateBulkAssetDataRowTest</name></test-case><test-case start="1444806797627" stop="1444806797632" status="passed"><name>ReadFileTest_Failure</name></test-case><test-case start="1444806795566" stop="1444806795576" status="passed"><name>MilesToKilometersTest</name></test-case><test-case start="1444806792836" stop="1444806792842" status="passed"><name>DoubleToSingleDigitStringTest</name></test-case><test-case start="1444806795544" stop="1444806795554" status="passed"><name>MaxTest</name></test-case><test-case start="1444806797978" stop="1444806797984" status="passed"><name>GenerateTest_Boundary</name></test-case><test-case start="1444806793466" stop="1444806793475" status="passed"><name>GetTimeZoneNameTest</name></test-case><test-case start="1444806797871" stop="1444806797879" status="passed"><name>ConfigureLoggingTest</name></test-case><test-case start="1444806785838" stop="1444806787765" status="passed"><name>ConvertToTypeStringTest</name></test-case><test-case start="1444806793513" stop="1444806793521" status="passed"><name>HexToIntTest</name></test-case><test-case start="1444806783679" stop="1444806785287" status="passed"><name>CreateDataReportTest</name></test-case><test-case start="1444806785596" stop="1444806785623" status="passed"><name>LoadFridgeDataTest_2</name></test-case><test-case start="1444806792935" stop="1444806792943" status="passed"><name>GetDateTimeTest</name></test-case><test-case start="1444806793493" stop="1444806793499" status="passed"><name>HaversineFormulaTest_NaN</name></test-case><test-case start="1444806792962" stop="1444806792970" status="passed"><name>GetFirmwareGenerationTest</name></test-case><test-case start="1444806787823" stop="1444806787835" status="passed"><name>CreateDebugFileTest</name></test-case><test-case start="1444806788519" stop="1444806788527" status="passed"><name>CreateCrcMoogTest</name></test-case><test-case start="1444806788539" stop="1444806790814" status="passed"><name>CreateKeyTest</name></test-case><test-case start="1444806793280" stop="1444806793363" status="passed"><name>GetInvalidControlColorTest</name></test-case><test-case start="1444806797993" stop="1444806797998" status="passed"><name>GenerateTest_ExcludeSymbols</name></test-case><test-case start="1444806788500" stop="1444806788507" status="passed"><name>CopyStreamTest_Failure_2</name></test-case><test-case start="1444806787843" stop="1444806787851" status="passed"><name>ExplodeDigitalSensorIndexTest</name></test-case><test-case start="1444806793545" stop="1444806793553" status="passed"><name>KilometersToMilesTest</name></test-case><test-case start="1444806788141" stop="1444806788146" status="passed"><name>GetTotalAlarmValuesTest_Failure</name></test-case><test-case start="1444806797634" stop="1444806797642" status="passed"><name>ReplaceUnSafeFileNameCharactersTest</name></test-case><test-case start="1444806788283" stop="1444806788312" status="passed"><name>CalculateMd5FileTest</name></test-case><test-case start="1444806788077" stop="1444806788083" status="passed"><name>GetMenuStructureDataTest_Failure</name></test-case><test-case start="1444806788420" stop="1444806788452" status="passed"><name>ConvertToDoubleTest</name></test-case><test-case start="1444806798022" stop="1444806798028" status="passed"><name>GenerateTest_Symbols</name></test-case><test-case start="1444806790816" stop="1444806790824" status="passed"><name>DbNullToDefaultTest</name></test-case><test-case start="1444806785626" stop="1444806785632" status="passed"><name>LoadFridgeDiagnosticDataTest_1</name></test-case><test-case start="1444806798702" stop="1444806798717" status="passed"><name>ToStringTest</name></test-case><test-case start="1444806797810" stop="1444806797825" status="passed"><name>AddCategoryTest</name></test-case><test-case start="1444806792862" stop="1444806792870" status="passed"><name>FindTimeZoneByIdTest</name></test-case><test-case start="1444806793421" stop="1444806793430" status="passed"><name>GetTimeFromDateTimeTest</name></test-case><test-case start="1444806798029" stop="1444806798041" status="passed"><name>AddEndSlashTest</name></test-case><test-case start="1444806787767" stop="1444806787792" status="passed"><name>CreateDataFileTest_1</name></test-case><test-case start="1444806793533" stop="1444806793543" status="passed"><name>JavaScriptUtcToDateTime</name></test-case></test-cases></test-suite>
```

**Comment by baev** [comment]
@ThaStealth thanks for suggestion.

The problem with your xml. You can validate it using this schema https://github.com/allure-framework/allure-core/blob/allure-core-1.4.19/allure-model/src/main/resources/allure.xsd

**Comment by ThaStealth** [comment]
@baev I read the comment in another thread about the invalid xml. But if I validate my xml to the xsd provided in your link by using http://www.freeformatter.com/xml-validator-xsd.html or http://www.utilities-online.info/xsdvalidation/ it states it is fully valid.

See: http://www.utilities-online.info/xsdvalidation/?save=b3a13a2b-2789-43fb-a383-47383c1769af-xsdvalidation

**Comment by baev** [comment]
@ThaStealth does you xml file has BOM? See http://stackoverflow.com/questions/4569123/content-is-not-allowed-in-prolog-saxparserexception

**Comment by ThaStealth** [comment]
@baev Yes it contains the following BOM: EF BB BF which represents an UTF8 file.

**Comment by baev** [comment]
@ThaStealth there is no build in way to deal with UTF-8 BOM in Java, so I don't think we should deal with it. Can you remove your BOM by yourself?

**Comment by ThaStealth** [comment]
@baev Ah, I didn't know that. The file was generated by the MSTestAllureAdapter. If I remove the BOM by hand it generates a report correctly! It seems the MSTestAllureAdapter sometimes generates files containing the BOM even when the UTF8Encoding is set to not to do this.

**Comment by baev** [comment]
> It seems the MSTestAllureAdapter sometimes generates files containing the BOM even when the UTF8Encoding is set to not to do this.

@ThaStealth could you please create issue for `MSTestAllureAdapter `?

**Comment by ThaStealth** [comment]
@baev  No problem, I'll create a ticket there.

Something else I noticed was that the Teamcity plugin collects the artifacts generated by Allure and puts them in the root directory of the artifacts. Since we configured teamcity to also put the .exe and .dll files of the application overthere it gets a little messy.
Can I configure the plugin to put it in a subdir in the artifacts output? The Reports tab seems to be able to handle this in Teamcity.

I don't know if you want me to create a new ticket for this?
